packages/indexer-cli/src/commands/indexer/rules/stop.ts

```diff
@@ -48,24 +47,19 @@ module.exports = {
 
     const config = loadValidatedConfig()
 
-    try {
-      validateDeploymentID(deployment, { all: false, global: true })
-    } catch (error) {
-      print.error(error.toString())
-      process.exitCode = 1
-      return
-    }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [identifier, identifierType] = await processIdentifier(id, { all: false, global: true })
 
     const inputRule = parseIndexingRule({
-      deployment,
+      identifier,
```

Comment:
might need to add `identifierType,` here and in some other `parseIndexingRule` such as `start,set,stop,maybe,clear`

Reply:
The identifier is a unique primary key, so the identifier type shouldn't be needed.  I'll make sure we have some tests for these commands.

Reply:
I got these when trying the functions without identifier type 
`[GraphQL] Variable "$rule" got invalid value { decisionBasis: "rules", identifier: "0xbbde25a2c85f55b53b7698b9476610c3d1202d88870e66502ab0076b7218f98a" }; Field "identifierType" of required type "IdentifierType!" was not provided.`

Reply:
o yep, the model requires them even though they aren't needed in those instances. I've updated these to fix and included tests for all. :)